test/plots/stackedAreaPlotTests.ts

```diff
@@ -1,486 +1,373 @@
+        stackedAreaPlot.y0(accessor);
+        assert.strictEqual(stackedAreaPlot.y0().accessor, accessor, `property set for datum`);
+        stackedAreaPlot.destroy();
+      });
     });
 
-    it("disables downsampling", () => {
-      assert.strictEqual(renderer.downsamplingEnabled(), false, "downsampling is disabled by default");
-      renderer.downsamplingEnabled(true);
-      assert.strictEqual(renderer.downsamplingEnabled(), false, "downsampling will not be enabled by user");
-      svg.remove();
+    describe("rendering on edge case scenarios", () => {
+      it("renders nothing when no data", () => {
+        let svg = TestMethods.generateSVG();
+        let stackedAreaPlot = new Plottable.Plots.StackedArea();
+        stackedAreaPlot.x(() => null);
+        // HACKHACK https://github.com/palantir/plottable/issues/2712 y scale must be set.
+        stackedAreaPlot.y(() => null, new Plottable.Scales.Linear());
+        stackedAreaPlot.renderTo(svg);
+        assert.strictEqual(stackedAreaPlot.selections().size(), 0, "no areas rendered");
+        stackedAreaPlot.destroy();
+        svg.remove();
+      });
+
+      it("coerces strings to numbers for calculating offsets", () => {
+        let svg = TestMethods.generateSVG();
+        let data0 = [
+          { x: 2, y: "2" },
+          { x: 3, y: 2 }
+        ];
+        let data1 = [
+          { x: 2, y: "0" },
+          { x: 3, y: "2" }
+        ];
+        let data2 = [
+          { x: 2, y: 2 },
+          { x: 3, y: 2 }
+        ];
+        let xScale = new Plottable.Scales.Linear();
+        let yScale = new Plottable.Scales.Linear();
+
+        let plot = new Plottable.Plots.StackedArea<number>();
+        let dataset0 = new Plottable.Dataset(data0);
+        plot.addDataset(dataset0);
+        let dataset1 = new Plottable.Dataset(data1);
+        plot.addDataset(dataset1);
+        let dataset2 = new Plottable.Dataset(data2);
+        plot.addDataset(dataset2);
+        plot.x((d: any) => d.x, xScale).y((d: any) => d.y, yScale);
+        plot.renderTo(svg);
+
+        let stackedAreaSelections = plot.content().selectAll("path");
+        let stackedAreaSelection0 = d3.select(stackedAreaSelections[0][0]);
+        let stackedAreaSelection1 = d3.select(stackedAreaSelections[0][1]);
+        let stackedAreaSelection2 = d3.select(stackedAreaSelections[0][2]);
+
+        let areaVertices0 = TestMethods.areaVertices(stackedAreaSelection0);
+        let areaYs0 = areaVertices0.map((areaVertex) => areaVertex.y).slice(0, -2);
+        let areaVertices1 = TestMethods.areaVertices(stackedAreaSelection1);
+        let areaYs1 = areaVertices1.map((areaVertex) => areaVertex.y).slice(0, -2);
+        let areaVertices2 = TestMethods.areaVertices(stackedAreaSelection2);
+        let areaYs2 = areaVertices2.map((areaVertex) => areaVertex.y).slice(0, -2);
+
+        assert.closeTo(areaYs0[0], yScale.scale(<any> data0[0].y), window.Pixel_CloseTo_Requirement,
```

Comment:
`data0[0]` is a string and our current behavior is such that there is a coercion done when stacking from strings to numbers.  In accordance there must be a coercion done from `string`s to `number`s for the scale